gm4_cement_mixers/data/cement_mixers/functions/item_fill.mcfunction

```diff
@@ -0,0 +1,19 @@
+#@s = liquid tank with item in first slot
+#run from liquid_tanks:item_process
+
+execute if score @s[tag=gm4_lt_water] gm4_lt_value matches 1.. if block ~ ~ ~ hopper{CustomName:"{\"text\":\"Water Tank\"}",Items:[{Slot:0b,id:"minecraft:black_concrete_powder"}]} run function cement_mixers:item_fill/black_concrete
```

Comment:
* `if score @s[tag=gm4_lt_water] gm4_lt_value matches 1..`
    * This should be done once and then call a second function with all the recipes.
* `CustomName:"{\"text\":\"Water Tank\"}"`
    * Is this really needed. I'm guessing this is the way it's done in LT itself but I don't like it.

Reply:
First point is a good one. Second one is something Special did in order to stop people breaking and hopper and replacing it with a differently named one really quickly (a bug someone found)

Reply:
That someone is me.
The bug was that there were no tags, and only the name is used. Currently both name and tag have to match.

I kinda told special about this after the bug fix. Again not a bug but could be changed. https://github.com/Gamemode4Dev/GM4_Datapacks/pull/105#r31746669

Reply:
Since the module has to examine block nbt for the item content anyways, it dosen't hurt to also check the name. This also helps to double-check that something funky hasn't gone on with the tank and that the user can always rely on the name of the tank to accurately reflect what they can get from it.